.github/workflows/check_s3.yml

```diff
@@ -0,0 +1,45 @@
+name: Check S3
+
+on:
+  schedule:
+  # every 8 hours, 10 minutes after the hour
+    - cron:  '10 */8 * * *'
+
+jobs:
+  check-s3-last-modified:
+    runs-on: ubuntu-latest
+    steps:
+    - id: repo-checkout
+      name: Check out this repo
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - id: install-aws
+      name: Install awscli
+      run: |
+        sh awscli_install.sh
```

Comment:
I would look into using an Action on the GitHub Marketplace for installing the cli, this looks promising https://github.com/marketplace/actions/install-aws-cli-action

One benefit of actions in general is that you can `use` actions made by the community to install various tools. If that works than the bash script wouldn't be needed.